docs/specs/sequences/endpoint_reconfiguration.md

```diff
@@ -57,6 +57,21 @@ sequenceDiagram
 
 ```
 
+### Commit Reconfiguration (Successful reconfiguration, persists Reboot)
+- Documentation on [device reboot mechanism through UDMI](../system_mode.md)
+
+```mermaid
+%%{wrap}%%
+sequenceDiagram
+    autonumber
+    participant D as Device
+    participant E as Original Endpoint
+    participant E' as New Endpoint
+    D->>E':STATE MESSAGE<br/>blobset.blobs._iot_endpoint_config.phase = "final"
+    note over D,E': Reboot sequence from new endpoint
+    D->>E':STATE MESSAGE<br/>blobset.blobs._iot_endpoint_config.phase = "final"
+```
+
```

Comment:
Maybe add a simple note after that just says "This just means the new endpoint connection is remembered after a system restart" I dunno -- it's one of those "it's so simple I'm not sure people will see it!" situations...